### PR TITLE
Dilpeet | Beta |  Pull from beta-branch in beta-stage new branch

### PR DESCRIPTION
**Pull request by dilpreet77**

### CODE REVIEW

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 1]
```diff
@@ -406,20 +465,100 @@ button.btn.btn-filter span.icon-cross {
...
+
+.customer-form-inline {
+    align-items: center;
+    width: 100.25%;
+    position: relative;
+    bottom: 2px;
+}
+
+.pending-approval-button {
+    background-color: #f8f9fa;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.scss:512]: use color variable

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 2]
```diff
@@ -406,20 +465,100 @@ button.btn.btn-filter span.icon-cross {
...
+    width: 140px;
+    height: 37px;
+}
+
+.pending-approval-button:hover {
+    background-color: #e2e6ea;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.scss:518]: use color variable

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 3]
```diff
@@ -681,11 +842,88 @@ button.btn.btn-filter span.icon-cross {
...
+    font-size: 25px;
+}
+.min-width-230 {
+    min-width: 230px;
+}
+.arrow-icon {
+    right: -3px;
+    width: 30px;
+    height: 30px;
+    border: 2px solid rgb(217, 217, 217);
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.scss:903]: use color variable

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 4]
```diff
@@ -681,11 +842,88 @@ button.btn.btn-filter span.icon-cross {
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.arrow-icon:hover {
+    color: white;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.scss:909]: use color variable

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 5]
```diff
@@ -681,11 +842,88 @@ button.btn.btn-filter span.icon-cross {
...
+    background-color: rgb(91, 100, 201);
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.scss:910]: use color variable

File: apps/web-giddh/src/app/theme/form-fields/text-field/text-field.component.html [hunk 6]
```diff
@@ -1,23 +1,17 @@
 <div class="text-field-container">
     <input
-        type="text"
-        *ngIf="useMaterial"
         matInput
         [name]="name"
+        [type]="type"
         [id]="id"
         [class]="cssClass"
+        [type]="type"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/form-fields/text-field/text-field.component.html:8]: remove this since [type] is already declared above.

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**